addon/controllers/search-project.js

```diff
@@ -26,14 +26,20 @@ export default class SearchProjectController extends Controller {
   }
 
   @action
-  selectProject(eproid) {
+  async selectProject(eproid) {
     if (this.model?.id) {
       const link = this.store.createRecord("gwr-link", {
         eproid,
         localId: this.model.id,
       });
-      link.save();
-      this.router.transitionTo("project.index");
+      try {
+        await link.save();
+        this.constructionProject.all.perform(this.model.id);
+      } catch {
+        await link.deleteRecord();
+        // Ignore, trying to link project that is already linked.
```

Comment:
Maybe add notification that link already exists.

Reply:
Done